tests/playTests.sh

```diff
@@ -588,6 +595,18 @@ if [ -n "$READFROMBLOCKDEVICE" ] ; then
     rm -f tmp.img tmp.img.zst tmp.img.copy
 fi
 
+println "\n===>  zstd created file timestamp tests"
+datagen > tmp
+sleep 1 # could also use touch -t but I don't know how portable that is.
```

Comment:
note : 
`sleep 1` adds 1 full second to this test.
This may not sound like much, but within the context of `playTests.sh`, we try to keep the test set "relatively" fast, and since there are a lot of tests to run , each test is supposed to only cost a fraction of a second.
When it doesn't, we generally move the test to the `--long-tests` category.

Possible mitigation : GNU `sleep` is supposed to accept floating point format.
So `sleep 0.01` would do.
However, I'm also not sure how portable that is.

Reply:
The comparison we're making is at 1-second resolution. :(

But maybe it's worth trying the `touch` route to set an artificial mtime.